src/index.js

```diff
@@ -1,38 +1,71 @@
 import './styles/style.css';
+/* eslint-disable */
+import { Task, addTask, removeTask, checkLocalStorage, editTask } from './module/utilityFunctions.js';
+import * as Elements from './module/constElements.js';
+/* eslint-enable */
```

Comment:
I was facing this problem I decided to make a Loom video to make explanation easier for you
- [ ] [OPTIONAL] Kindly add easier way to exit edit mode and save changes
- [ ] Let's kindly make sure we have our checkbox working as intended 

https://www.loom.com/share/a04e28b431554525a845c2d58fe36309

Reply:
@aboongm  Yes you can leave the edit as-is for now
enabling eslint is preferred but I don't want to overload you with other tasks when you are still working on specific ones and having difficulty It would be awesome if you fix/learn from eslint